### PR TITLE
 Changes for missing translations in selector

### PR DESCRIPTION
**Pull request by vishesh-ats**
[CC-78](https://github.com/Components-care/backlog/issues/78)

### CODE REVIEW

File: src/assets/i18n/de/translation.json [hunk 1]
```diff
@@ -60,7 +60,13 @@
 			}
 		},
 		"selector": {
-			"add-new": "Hinzufügen"
+			"add-new": "Hinzufügen",
+			"base-selector": {
+				"no-options-text": "Keine Optionen",
+				"close-icon-text": "Schließen",
+				"open-icon-text": "Öffnen Sie",
```
Comment (TQjona) [inline src/assets/i18n/de/translation.json:67]: should be `Öffnen`

File: src/assets/i18n/de/translation.json [hunk 2]
```diff
@@ -60,7 +60,13 @@
...
+				"loading-text": "Laden..."
```
Comment (TQjona) [inline src/assets/i18n/de/translation.json:68]: should be `Lade...`

**Review by TQjona — CHANGES_REQUESTED** [review]
MultiSelectWithoutGroup needs to be localized too

**Comment by YvesR** [comment]
@vishesh-ats @ankitp-atharva @TQjona 
For translation I prefer you 
- create all files if not exists
- translate only English
- inform me to translate

Often I even want to change English translation and in this case I can adjust and translate.

**Comment by vishesh-ats** [comment]
@TQjona , Changes updated. Please review.

**Review by TQjona — APPROVED** [review]

**Review by ankitp-atharva — APPROVED** [review]

**Comment by vishesh-ats** [comment]
@YvesR , Please review newly added locale here. If all good then we can merge this PR.

**Pull request merged**